src/client/src/apps/SpotlightRoute/Spotlight.tsx

```diff
@@ -109,11 +109,13 @@ export const Spotlight: FC = () => {
       <Response>
         {contacting ? (
           <>
-            <AdaptiveLoader size={14} speed={0.8} seperation={1.5} type="secondary" />
+            <AdaptiveLoader size={14} speed={0.8} seperation={1.5} type="secondary"/>
             <Contacting>Contacting…</Contacting>
           </>
         ) : (
-          response
+          <Suggestion onClick={() => handleIntent(response, platform)}>
```

Comment:
as i understand you want user to explicitly select an intent, is this correct? wouldn't it be more "responcive" if we handle intent straightway? (at least when we know there's a single intent)